src/core/regimes.rkt

```diff
@@ -45,7 +71,7 @@
   (define repr (context-repr ctx))
   (timeline-push! 'repr (~a (representation-name repr)))
   (timeline-push! 'oracle (errors-score (map (curry apply max) err-lsts)))
-  best)
+  (values best best-index errs))
```

Comment:
The pretty complicated API here is another reason I think `pareto-regimes` should be moved into `regimes`, and then only it should be provided.

Reply:
The problem is that `pareto-regimes` uses `combine-alts` which, according to the comments in `regimes.rkt`, is in `bsearch.rkt` so that mainloop can insert a timeline break. If I put `pareto-regimes` in `regimes.rkt`, I'll have to move `combine-alts` into `regimes.rkt` as well, which I assume would break this timeline break. Not sure how important this timeline break is.

Reply:
Ok, got it. You can do this in a separate PR if you'd like, but it sounds to me like we actually want `pareto-regimes` to run and return a list of `option`s, which the `mainloop` would then pass one by one to `combine-alts`. This keeps the timeline break and is neater in general. Again, could be this PR or could be another one if it sounds too messy.